src/trainer_see.c

```diff
@@ -447,7 +448,7 @@ static u8 CheckTrainer(u8 objectEventId)
         struct ScriptContext ctx;
         if (RunScriptImmediatelyUntilEffect(SCREFF_V1 | SCREFF_SAVE | SCREFF_HARDWARE | SCREFF_TRAINERBATTLE, trainerBattlePtr, &ctx))
         {
-            if (*ctx.scriptPtr == 0x5c) // trainerbattle
+            if (*ctx.scriptPtr == SCR_OP_TRAINERBATTLE)
```

Comment:
Can you please replace the magic numbers in `src/vs_seeker.c`?

`if (script[0] == 0x5c)` and `else if (script[0] == 0x23)`